webapi/src/test/scala/org/knora/webapi/config/AppConfigSpec.scala

```diff
@@ -0,0 +1,27 @@
+package org.knora.webapi.config
+
+import zio.test.Assertion._
+import zio.test.TestAspect.ignore
+import zio.test.TestAspect.timeout
+import zio.test._
+import zio._
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.FiniteDuration
+
+object AppConfigSpec extends ZIOSpec[AppConfig] {
+
+  val layer = ZLayer.make[AppConfig](AppConfig.live)
+
+  def spec = suite("ApplicationConfigSpec")(
+    test("successfully provide the application configuration") {
+      for {
+        appConfig <- ZIO.service[AppConfig]
+        // _         <- ZIO.debug(appConfig)
+      } yield {
+        assertTrue(appConfig.printExtendedConfig == false)
+        assertTrue(appConfig.jwtLongevityAsDuration == FiniteDuration(30L, TimeUnit.DAYS))
+        assertTrue(appConfig.sipi.timeoutInSeconds == FiniteDuration(120L, TimeUnit.SECONDS))
+      }
```

Comment:
is this generally how we write tests now?

Reply:
it is one way of doing it that works. I don't know yet what the canonical way is. There is no documentation describing it. But this looks clean to me.  The assertions could also be part of the `for` comprehension. I guess, it depends on the test.